docs/desktop-feature-api.mdx

```diff
@@ -76,35 +104,83 @@ pref("browser.aboutwelcome.enabled", true);
 
 Import the `NimbusFeatures` module:
 
+<Tabs
+  defaultValue="js"
+  values={[
+    { label: "JavaScript", value: "js" },
+    { label: "C++", value: "cpp" },
+  ]
+}>
+<TabItem value="js">
+
 ```js
 XPCOMUtils.defineLazyModuleGetters(this, {
   NimbusFeatures: "resource://nimbus/ExperimentAPI.jsm",
 });
 ```
 
+</TabItem>
+<TabItem value="cpp">
+
+```cpp
+#include "mozilla/browser/NimbusFeatures.h"
+```
+</TabItem>
+</Tabs>
+
 ## API Reference Guide
 
 ### `isEnabled()`
 
-`isEnabled({sendExposureEvent = false}): boolean`
+`isEnabled(): boolean`
 
 Short form for `.getVariable("enabled")`
 
 Note that you should have an `enabled` property defined in the manifest if you plan to use this.
 
+<Tabs
+  defaultValue="js"
+  values={[
+    { label: "JavaScript", value: "js" },
+    { label: "C++", value: "cpp" },
+  ]
+}>
+
+<TabItem value="js">
+
 ```js
 const isEnabled = NimbusFeatures.myFeature.isEnabled();
 ```
 
+</TabItem>
+<TabItem value="cpp">
+
+```cpp
```

Comment:
Can you mark the API methods that are JS only some way? Maybe like in the type definition line:

```
`isEnabled(): boolean` (JS Only)
```